docs/UserGuide.md

```diff
@@ -38,7 +38,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Features
```

Comment:
Any reason for this formatting choice? Just afraid will cause random merge conflicts, haha

Reply:
Yup, it's because I'm using ## to display the section header 'Event Management', so I upgraded Features to a single #. For each individual command we are currently using ###.